data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1918 @@
+			choice
+				`	"Alright, I'll go there once I have an outfit scanner ready."`
+				`	"That sounds even more dangerous than what you had me do before. Sorry, but I'm not interested."`
+					decline
+			`	They thank you for your willingness to help, and tell you they will be waiting on the <destination>.`
+				accept
+	on complete
+		require "Outfit Scanner"
+
+mission "Heliarch Recon 2-B"
+	name "Silent Scan"
+	description "Now that the Heliarchs have altered your Outfit Scanner, head to <waypoints> and scan some Quarg ships there, then report back to <destination>."
+	landing
+	source "Ring of Wisdom"
+	waypoint "Enif"
+	to offer
+		has "Heliarch Recon 2-A: done"
+		not "Resistance: Questions: done"
+	on offer
+		conversation
+			`The Heliarch agents who spoke to you are ready with a team of engineers once you land. They remove the outfit scanner from your ship and take it to some restricted area. After about an hour, they come back, and while you can't see any visible changes to the outfit, they assure you it is better suited for scanning Quarg ships now.`
+			`	The agents come into your ship, asking for your map again. "Now, in the earliest stages of construction, this ring you showed us is," the Arach says, pointing to <waypoints>, where Lagrange is. "Interested in the Quarg ships guarding that system, Consul Aulori is. Suffice, one scan per type of ship there should."`
+				accept
+	npc "scan outfits"
+		personality staying uninterested target
+		government "Quarg"
+		system "Enif"
+		fleet
+			names "quarg"
+			variant
+				"Quarg Skylark"
+				"Quarg Wardragon"
+	on complete
+		"coalition jobs" ++
+		payment 200000
+		conversation
+			`Once again the engineers remove your outfit scanner, taking a copy of the scan logs and handing it to the Heliarch agents, who give you <payment>. "Grateful to you, we are, Captain <last>," the Saryd says. "Remember your assistance, the consul will."`
+			`	When the engineers return, they put your outfit scanner back in place. The Heliarch agents say their goodbyes, reassuring you that the Quarg shouldn't attack you for this. "If interested in doing some investigation of your own, you wish, maybe land on their rings again, you could," the Kimek suggests. "Maybe find out something of interest for us, you will."`
+			`	He wishes you safe travels, and leaves with his colleagues.`
+
+mission "Quarg Interrogation"
+	invisible
+	landing
+	deadline 1
+	source
+		government "Quarg"
+		not planet "Humanika"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 2-B: done"
+	on offer
+		conversation
+			`As you land on <origin>, a Quarg is patiently waiting right outside. Seeing that it won't go away, you head outside to speak with it.`
```

Comment:
I think this could be expanded. Are you just sitting inside, waiting for them to leave, or do you try something? Does the Quarg make some gesture for you to come out, or that they won't leave?